aerospike/README.md

```diff
@@ -67,7 +71,7 @@ For containerized environments, see the [Autodiscovery Integration Templates][5]
 | -------------------- | ------------------------------------ |
 | `<INTEGRATION_NAME>` | `aerospike`                          |
 | `<INIT_CONFIG>`      | blank or `{}`                        |
-| `<INSTANCE_CONFIG>`  | `{"host":"%%host%%", "port":"3000"}` |
+| `<INSTANCE_CONFIG>`  | `{"openmetrics_endpoint": "http://127.0.0.1:9145/metrics"}` |
```

Comment:
Just realized, this should probably be `{"openmetrics_endpoint": "http://%%host%%metrics"}`